src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -1104,7 +1108,11 @@ void ZapInfo::allocMem(
 
     if (roDataSize > 0)
     {
-        if (flag & CORJIT_ALLOCMEM_FLG_RODATA_16BYTE_ALIGN)
+        if (flag & CORJIT_ALLOCMEM_FLG_RODATA_32BYTE_ALIGN)
```

Comment:
I would not bother changing anything for zapper. We are trying to churn the old crossgen and there are number of other changes to make it actually work (e.g. in NewAlignedBlob).